app/models/katello/content_view_environment.rb

```diff
@@ -26,11 +29,16 @@ class ContentViewEnvironment < Katello::Model
     scope :non_default, -> { joins(:content_view).where("katello_content_views.default" => false) }
     scope :default, -> { joins(:content_view).where("katello_content_views.default" => true) }
     alias :lifecycle_environment :environment
+    delegate :organization, :to => :environment
```

Comment:
Since this is actually an association and not just a method, what are your thoughts on this?

```suggestion
    has_one :organization, :through => :environment
```

https://guides.rubyonrails.org/association_basics.html#the-has-one-through-association